### PR TITLE
Chore/sergio q/docu

### PR DESCRIPTION
**Pull request by sergioqfeg1**
Documentation for the part 5

### CODE REVIEW

File: docs/src/05_building_block_view.adoc [hunk 1]
```diff
@@ -39,43 +41,37 @@ Thus it contains the white box description of selected building blocks of level
 See https://docs.arc42.org/section-5/[Building Block View] in the arc42 documentation.
 
 ****
+////
+
 
 === Whitebox Overall System
 
 [role="arc42help"]
```
Comment (Toto-hitori) [inline docs/src/05_building_block_view.adoc:49]: Remove this line please

File: docs/src/05_building_block_view.adoc [hunk 2]
```diff
@@ -39,43 +41,37 @@ Thus it contains the white box description of selected building blocks of level
...
-Since there are so many ways to specify interfaces why do not provide a specific template for them.
- In the worst case you have to specify and describe syntax, semantics, protocols, error handling,
- restrictions, versions, qualities, necessary compatibilities and many things more.
-In the best case you will get away with examples or simple signatures.
-
+This is the overall view of the application. The diagram is composed of 3 elements that will interact between each other.
 ****
 
-_**<Overview Diagram>**_
+_**Overview Diagram**_
```
Comment (Toto-hitori) [inline docs/src/05_building_block_view.adoc:54]: This must either be a header or be removed

File: docs/src/05_building_block_view.adoc [hunk 3]
```diff
@@ -39,43 +41,37 @@ Thus it contains the white box description of selected building blocks of level
...
+image::BusinessContext.png["Overall view of the business context"]
 
 Motivation::
+This will be the general sketch of the elements interacting inside the application, including the external elements that will include the application.
 
-_<text explanation>_
+Contained Building Blocks::
+****
 
+* **Player:** This is the user that will be playing with our application. They will need tl be authenticated to be able to play.
```
Comment (Toto-hitori) [inline docs/src/05_building_block_view.adoc:64]: They will need ***to*** be

File: docs/src/05_building_block_view.adoc [hunk 4]
```diff
@@ -39,43 +41,37 @@ Thus it contains the white box description of selected building blocks of level
...
-_<Description of contained building block (black boxes)>_
+****
 
 Important Interfaces::
-_<Description of important interfaces>_
+This part will be more detailed later, since the structure of the different interfaces/classes has not been discussed by the team yet.
+
 
+////
 [role="arc42help"]
```
Comment (Toto-hitori) [inline docs/src/05_building_block_view.adoc:75]: Remove this

File: docs/src/05_building_block_view.adoc [hunk 5]
```diff
@@ -143,28 +139,43 @@ _<black box template>_
...
-Here you can specify the inner structure of (some) building blocks from level 1 as white boxes.
-
-You have to decide which building blocks of your system are important enough to justify such a detailed description.
-Please prefer relevance over completeness. Specify important, surprising, risky, complex or volatile building blocks.
-Leave out normal, simple, boring or standardized parts of your system
+Here is an specification of the inner structure of the WIQ Application.
 ****
 
-==== White Box _<building block 1>_
+==== White Box _<WIQ Application>_
```
Comment (Toto-hitori) [inline docs/src/05_building_block_view.adoc:153]: Remove the <> since they appear in the title.

File: docs/src/05_building_block_view.adoc [hunk 6]
```diff
@@ -143,28 +139,43 @@ _<black box template>_
...
+
+image::ContainerDiagram.png["Container for the WIQ System"]
 
 [role="arc42help"]
```
Comment (Toto-hitori) [inline docs/src/05_building_block_view.adoc:157]: Remove this

File: docs/src/05_building_block_view.adoc [hunk 7]
```diff
@@ -143,28 +139,43 @@ _<black box template>_
...
 ****
-...describes the internal structure of _building block 1_.
+This diagram describes the internal organization of the WIQ Application.
 ****
 
-_<white box template>_
+//_<white box template>_
```
Comment (Toto-hitori) [inline docs/src/05_building_block_view.adoc:162]: remove this

**Review by Toto-hitori — COMMENTED** [review]
Remove this, the rest is ok

**Review by gony02 — APPROVED** [review]

**Review by Toto-hitori — APPROVED** [review]

**Review by UO283615 — APPROVED** [review]

**Pull request merged**